--- test/bash/64.out	2025-10-01 16:17:32.731162617 +0100
+++ test/mini/64.out	2025-10-01 16:17:32.735162660 +0100
@@ -0,0 +1 @@
+idontexist: command not found
--- test/bash/64.err	2025-10-01 16:17:32.735162660 +0100
+++ test/mini/64.err	2025-10-01 16:17:32.735162660 +0100
@@ -1 +0,0 @@
-bash: line 1: idontexist: command not found
--- test/bash/64.status	2025-10-01 16:17:32.735162660 +0100
+++ test/mini/64.status	2025-10-01 16:17:32.735162660 +0100
@@ -1 +1 @@
-127
+1
